app/scripts/modules/core/chaosMonkey/chaosMonkeyExceptions.component.spec.ts

```diff
@@ -33,13 +33,13 @@ describe('Controller: ChaosMonkeyExceptions', () => {
 
     it('gets all accounts, then adds wildcard and regions per account to vm', () => {
       let accounts = [ {name: 'prod'}, {name: 'test'} ];
-      let details = {
+      let details: any = {
         prod: { name: 'prod', regions: [ {name: 'us-east-1'}, {name: 'us-west-1'}] },
         test: { name: 'test', regions: [ {name: 'us-west-2'}, {name: 'eu-west-1'}] }
       };
 
       spyOn(accountService, 'listAccounts').and.returnValue($q.when(accounts));
-      spyOn(accountService, 'getAccountDetails').and.callFake((accountName) => {
+      spyOn(accountService, 'getAccountDetails').and.callFake((accountName: string): ng.IPromise<any> => {
```

Comment:
did the linter complain here about a missing return type on an arrow function?  i forgot to bring that up this morning.  the general thought by the community was to have them on arrow function arguments, but not a return type.